mlflow/tracking/llm_utils.py

```diff
@@ -6,12 +6,12 @@
 
 import mlflow
 from mlflow.tracking.client import MlflowClient
-from mlflow.utils.annotations import experimental
+from mlflow.utils.annotations import deprecated
 
 _logger = logging.getLogger(__name__)
 
 
-@experimental
+@deprecated("mlflow.log_table")
```

Comment:
do we want to indicate the removal flag? It's been marked as experimental, so I'm not sure if you think it's needed or not. 

`@deprecated("mlflow.log_table", since="2.8.1", impact=True)` ?

Reply:
Done! :)